src/mmw/js/src/modeling/models.js

```diff
@@ -288,7 +288,8 @@ var ScenarioModel = Backbone.Model.extend({
                     name: 'precipitation',
                     value: 1.0
                 }
-            ]
+            ], // Setting getResults to false can be useful when testing.
```

Comment:
Consider making this comment more clear that this is only here for unit testing purposes.